### ISSUE TITLE
add to usb sticks, 512 MB sticks (for windows xp computer) and more gpu!!!

### ISSUE THREAD

**Issue by vovaluzan** [Delta2Force/MCVmComputers#64]
Gpu 1 - 2d, 64 mb
Gpu 1.1 - 3d, 64 mb
Gpu 2 - 2d, 128 mb
Gpu 2.1 - 3d, 128 mb
Gpu 3 - 3d, 256 mb
Usb stiks
Usb stick 1 - 4 gb
Usb stick 2 - 8 gb
Usb stick 3 - 16 gb
Usb stick 4 - 32 gb

**Comment by Delta2Force** [comment]
The RAM sticks were added in 1.4.10.

**Comment by pivinx1** [comment]
I don't think that USB sticks can be added because VBox doesn't support creating virtual ones yet.

**Comment by BlueJay113** [comment]
> I don't think that USB sticks can be added because VBox doesn't support creating virtual ones yet.

VirtualBox supports the temporary creation of USB images. Once the VM closes, VirtualBox saves the changes to the real USB from the temporary USB image.

**Comment by pivinx1** [comment]
Alright, Hood to mnie. But I think that no one would want an temporary USB
stick.

sob., 19 wrz 2020, 23:19 użytkownik BlueJay113 <notifications@github.com>
napisał:

**Comment by BlueJay113** [comment]
It's not temporary though. The temporary image of the USB is a virtual bridge between VirtualBox and the USB itself.

**Comment by kristibektashi** [comment]
Actually you could use a life hack for USB support. Instead of using real USB drives, how about using virtual hard disks? There should be a USB item in-game with the capacities mentioned above and it should work similar to hard drives. When you right-click it it should automatically create a vhd (or vdi) in a different folder that hard drives (usbs instead of vhds) with the capacity that you bought (unlike hard drives, usb drives should have a set capacity (the ones in the original issue) and you cannot change it unless you buy another USB from the in-game tablet

**Comment by pivinx1** [comment]
I agree. I saw that you can set USB as the disk controller, so in theory
there is an possibility that USB drived can be made this way.

pt., 29 sty 2021, 22:17 użytkownik Kristibek <notifications@github.com>
napisał:
